operator/v1/stable.network.testsuite.yaml

```diff
@@ -15,3 +15,182 @@ tests:
+              routingViaHost: false
+        disableNetworkDiagnostics: false
+        logLevel: Normal
+        operatorLogLevel: Normal
+  - name: Should be able to pass a valid shorthand IPV6 CIDR to v6InternalMasqueradeSubnet
+    initial: | 
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec:
+       defaultNetwork:
+          ovnKubernetesConfig:
+            gatewayConfig:
+              ipv6:
+                v6InternalMasqueradeSubnet: "abcd:ef01::2345:6789/20"
+    expected: | 
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec:
+        defaultNetwork:
+          ovnKubernetesConfig:
+            gatewayConfig:
+              routingViaHost: false
+              ipv6:
+                v6InternalMasqueradeSubnet: "abcd:ef01::2345:6789/20"
+        disableNetworkDiagnostics: false
+        logLevel: Normal
+        operatorLogLevel: Normal
+  - name: Should not be able to pass invalid IPV6 CIDR to v6InternalMasqueradeSubnet
+    initial: | 
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec:
+        defaultNetwork:
+          ovnKubernetesConfig:
+            gatewayConfig:
+              ipv6:
+                v6InternalMasqueradeSubnet: "foo"
+    expectedError:  "Invalid value: \"string\": subnet must be in the range /0 to /125 inclusive"
+  - name: Should not be able to add an IP address with the incorrect number of octets to v6InternalMasqueradeSubnet
+    initial: | 
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec:
+        defaultNetwork:
+          ovnKubernetesConfig:
+            gatewayConfig:
+              ipv6:
+                v6InternalMasqueradeSubnet: abcd:ef01:2345:6789:abcd:ef01:2345:6789:abcd/125
+    expectedError: "Invalid value: \"string\": a valid IPv6 address must contain at most 8 segments, or at most 6 segments for a dual address"
+  - name: Should not be able to add a dual IP address with the incorrect number of octets to v6InternalMasqueradeSubnet
+    initial: | 
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec:
+        defaultNetwork:
+          ovnKubernetesConfig:
+            gatewayConfig:
+              ipv6:
+                v6InternalMasqueradeSubnet: abcd:ef01:2345:6789:abcd:ef01:2345:1.2.3.4/125
+    expectedError: "Invalid value: \"string\": a valid IPv6 address must contain at most 8 segments, or at most 6 segments for a dual address"
+  - name: Should be able to pass a double shorthand IPV6 CIDR to v6InternalMasqueradeSubnet
+    initial: | 
+      apiVersion: operator.openshift.io/v1
+      kind: Network
+      spec:
+       defaultNetwork:
+          ovnKubernetesConfig:
+            gatewayConfig:
+              ipv6:
+                v6InternalMasqueradeSubnet: "abcd::ef01::2345:6789/20"
+    expectedError: "Invalid value: \"string\": IPv6 addresses must contain at most one '::' and may only be shortened once"
```

Comment:
Nit, all files should end with a new line character, git can produce weird diffs otherwise